modules/react/common/stories/Responsive.stories.mdx

```diff
@@ -0,0 +1,241 @@
+import {Meta, ArgsTable} from '@storybook/addon-docs';
+import {Box} from '@workday/canvas-kit-react/layout';
+import {useResizeObserver, useResponsiveContainerStyles} from '@workday/canvas-kit-react/common';
+import {ResponsiveContainer} from './examples/ResponsiveStyles';
+import {ResponsiveViewport} from './examples/ResponsiveViewport';
+
+<Meta title="Hooks and Utilities/Responsive Styling" />
+
+# Responsive Styling
+
+Responsive styling is an approach to web development that adjusts layout and appearance based on the
+device's viewport size or an element's container size.
+
+## What is Responsive Styling?
+
+Responsive Styling it is a term used to describe an approach to web design or a set of best
+practices, used to create a layout that can _respond_ to the device being used to view the content.
```

Comment:
We don't need that paragraph under the H1 since it'll just be stripped out on the site. Moving it underneath `## What is Responsive Styling?`.
```suggestion
## What is Responsive Styling?

Responsive styling is an approach to web development that adjusts layout and appearance based on the
device's viewport size or an element's container size.
```